src/index.js

```diff
@@ -122,28 +110,16 @@ module.exports = {
     'prefer-destructuring': [
       'error',
       {
-        AssignmentExpression: {
-          array: false,
-          object: false
-        },
-        VariableDeclarator: {
-          array: true,
-          object: true
-        }
+        AssignmentExpression: { array: false, object: false },
+        VariableDeclarator: { array: true, object: true }
       },
       {
         enforceForRenamedProperties: false
       }
     ],
     'prefer-spread': 'error',
     'prefer-template': 'error',
-    'prettier/prettier': [
-      'error',
-      {
-        printWidth: 120,
-        singleQuote: true
-      }
-    ],
+    'prettier/prettier': ['error', { printWidth: 120, singleQuote: true, trailingComma: 'none' }],
```

Comment:
In Prettier, `arrowParens` was introduced in v1.9 and in v2.0 its value changed from `avoid` to `always`. Should we use the `avoid` instead? [source](https://prettier.io/docs/en/options.html#arrow-function-parentheses).

Reply:
Hmm, I'm having trouble getting prettier to ignore the parenthesis, seems like the option is not working..